kube/src/main/java/org/openshift/ping/kube/KubePing.java

```diff
@@ -195,8 +195,8 @@ public void destroy() {
         try {
             pods = client.getPods(_namespace, _labels);
         } catch (Exception e) {
-            if (log.isWarnEnabled()) {
-                log.warn(String.format("Problem getting Pod json from Kubernetes %s for cluster [%s], namespace [%s], labels [%s]; encountered [%s: %s]",
+            if (log.isTraceEnabled()) {
```

Comment:
The only thing required for this to fail is for the service account to not have view permissions on the cluster.  These permissions can be changed at any time.  For example, you will see these errors continuously until you add the view permission to the service account.  After that, the cluster will form correctly.  Just image the reverse case and you can see how this could fail after a successful cluster formation.